### DISCUSSION TITLE
小米空调伴侣2回报homekit环境温度问题

### DISCUSSION THREAD

**Discussion by 521tonysteven** [XiaoMi/ha_xiaomi_home#337] [Ideas]
问题描述：
我是用的是小米空调伴侣2，由于小米空调伴侣2没有温度传感器导致实体属性“current_temperature”值为null。回报到homekit时因“current_temperature”值为null，homekit会使用默认值21填充。
需求建议：
希望可以自定义空调伴侣2的实体属性，可以将米家温湿度传感器的温度值绑定到空调伴侣2实体上。参见[hass-xiaomi-miot](https://github.com/al-one/hass-xiaomi-miot)项目，此项目可以在customize.yaml中使用bind_sensor属性将可用温度传感器绑定到小米空调伴侣2实体上来解决此问题
![image](https://github.com/user-attachments/assets/afbf2096-edd0-48a7-aa77-3947e78952da)

**Comment by JimmyKmi** [comment]
同
https://github.com/XiaoMi/ha_xiaomi_home/issues/137
https://github.com/XiaoMi/ha_xiaomi_home/discussions/279
